examples/magento/getProductStock.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Denner\Client\MagentoClient;
+
+$config = require realpath(__DIR__ . '/../bootstrap.php');
+$params = array(
+);
+
+$client = MagentoClient::factory($config);
+
+$token = @$_GET['token'] ?: 'ajs8vvnm0vgk976iiqv6optdsjucp05r';
```

Comment:
Does it make sense to have a default token here? I recommend requiring it as URL param.